src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -245,6 +245,19 @@ private[spark] object Config {
       .stringConf
       .createWithDefaultString(DEFAULT_MEM)
 
+  val ARMADA_RUN_AS_USER: OptionalConfigEntry[Long] =
+    ConfigBuilder("spark.armada.runAsUser")
+      .doc("Specify the numeric id of user to run job as")
```

Comment:
What kind of user? OS user? Armada user?

Reply:
This is the Unix / OS user running inside the docker container.